src/incubator/WheelPicker/__tests__/index.spec.js

```diff
@@ -0,0 +1,70 @@
+import _ from 'lodash';
+import React from 'react';
+import {WheelPicker} from '../../../incubator';
+import {render} from '@testing-library/react-native';
+import {FireOnMomentumScrollEnd} from '../../../uilib-test-renderer';
+
+const ITEM_HEIGHT = 50;
+const NUM_OF_ROWS = 7;
+const onChangeMock = jest.fn();
+
+const TestCase = props => {
+  return (
+    <WheelPicker
+      testID={'wheel'}
+      items={_.times(60, i => i).map(item => ({label: `${item}`, value: item}))}
+      onChange={onChangeMock}
+      numberOfVisibleRows={NUM_OF_ROWS}
+      itemHeight={ITEM_HEIGHT}
+      initialIndex={0}
+      activeTextColor={'red'}
+      inactiveTextColor={'blue'}
+      {...props}
+    />
+  );
+};
+
+describe('WheelPicker', () => {
+  describe('FlatList', () => {
+    it('should call onChange callback after scrolling', () => {
+      const {getByTestId} = render(<TestCase/>);
+      const flatList = getByTestId('wheel.list');
+
+      FireOnMomentumScrollEnd(flatList, {y: 200});
+
+      expect(onChangeMock).toHaveBeenCalled();
+      expect(onChangeMock).toHaveBeenCalledWith(4, 4);
+
+      FireOnMomentumScrollEnd(flatList, {y: 330});
+
+      expect(onChangeMock).toHaveBeenCalledWith(7, 7);
+    });
+
+    it('should present 7 rows', () => {
+      const {getByTestId} = render(<TestCase/>);
+      const flatList = getByTestId('wheel.list');
+
+      expect(flatList.props.height).toBe(NUM_OF_ROWS * ITEM_HEIGHT);
+    });
+  });
+
+  //TODO: make this test work
+  //   describe('Item', () => {
+  //     it('should color selected index with activeColor', () => {
```

Comment:
This description is not very clear on what you want to test (where is the verb in this sentence?)